core/src/test/java/org/apache/accumulo/core/conf/PropertyModificationUtilTest.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.accumulo.core.conf;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import java.util.ConcurrentModificationException;
+import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.concurrent.atomic.AtomicInteger;
+
+import org.junit.jupiter.api.Test;
+
+public class PropertyModificationUtilTest {
+
+  // make sure exception is thrown after retry attempts exceeded
+  @Test
+  public void testConcurrentModificationExceptionPropagates() throws Exception {
+    try {
```

Comment:
I think the Junit5 way of doing this is to use `assertThrows`, see [example](https://github.com/apache/accumulo/blob/main/server/base/src/test/java/org/apache/accumulo/server/ServerDirsTest.java#L115)

Reply:
Thanks for the tip, I fixed this.